### PR TITLE
Document custom error page for boot errors

### PR DESCRIPTION
**Pull request by sofietoft**
Related to: https://github.com/umbraco/Umbraco-CMS/pull/9531 

Also cleaned up the "Custom Error Pages" tutorial and added a "Table of contents".

### CODE REVIEW

File: Tutorials/Custom-Error-Pages/index.md [hunk 1]
```diff
@@ -82,9 +92,22 @@ With this approach, you can set different 404 pages for different languages (cul
...
+Sometimes you might experience issues with booting up your Umbraco project. This could be a brand new project, or it could be an existing project after an upgrade.
+
+When there is an error during boot you will presented with a generic error page.
+
+![Boot Failed. Umbraco failed to boot, if you are the owner of the website please see the log file for more details.](images/BootFailedGeneric.png "Screen shot of generic BootFailed page")
+
+The file used for rendering this error page can be found here `~/umbraco/views/errors/BootFailed.html`.
+
+In order to customize this error page it is recommend that you create a **new HTML file** in `~/config/errors/` using the name `BootFailed.html`.
+
```
Comment (rasmusjp) [inline Tutorials/Custom-Error-Pages/index.md:106]: I think adding a note about the `BootFailed.html` page is only shown when `debug=false` would be great, something like this

```suggestion
:::note
The `BootFailed.html` page will only be shown if debugging is disabled in `web.config` i.e. `<compilation debug="false" />`.  The full error can always be found in the log file.
:::
```

**Review by rasmusjp — CHANGES_REQUESTED** [review]
Looks good 💪

I've left a single suggestion, what do you think?

**Comment by sofietoft** [comment]
Great catch @rasmusjp ! I completely forgot to mention that 💪 

I've committed the note. Thanks!

**Comment by rasmusjp** [comment]
Awesome 😄 I've approved the changes. I figure you are going to merge it yourself when 8.12 is released.

**Pull request merged**